Controllers/feedbackController.js

```diff
@@ -14,14 +14,14 @@ const {
 const logger = require("../utility/logger/logger");
 const redisClient = require("../utility/redisCaching/redisCache");
 const redisFunctions = require("../utility/redisCaching/redisFunctions");
+const getToken = require("../utility/getToken/getToken");
 
 /*This controller allows the interns to request for feedback using the request
 feedback forms.
  */
 
 const submitFeedBack = async (req, res) => {
-  const { authToken } = req.cookies;
-  const { id } = jwt.verify(authToken, process.env.JWT_SECRET);
+  const [id, username] = getToken(req);

```

Comment:
We only need id here and not username. Once you tweak your getToken function, you can adjust this. We don't want to get the username where we don't need it